packages/react-form/src/hooks/list/baselist.ts

```diff
@@ -53,11 +53,9 @@ export function useBaseList<Item extends object>(
   validationDependencies: unknown[] = [],
 ): BaseList<Item> {
   const list = Array.isArray(listOrConfig) ? listOrConfig : listOrConfig.list;
-  const validates: FieldListConfig<Item>['validates'] = Array.isArray(
-    listOrConfig,
-  )
-    ? {}
-    : listOrConfig.validates || {};
+  const validates: FieldListConfig<Item>['validates'] = useMemo(() => {
```

Comment:
Why memoize this? This doesn't look like an expensive operation at all.

Reply:
Linter wants it to be I can ignore but then we should look at the rule

Reply:
```
The 'validates' conditional could make the dependencies of useMemo Hook (at line 80) change on every render. Move it inside the useMemo callback. Alternatively, wrap the initialization of 'validates' in its own useMemo() Hook.eslintreact-hooks/exhaustive-deps
```

I think the linter is correct.